releasenotes/notes/0.5/0_5_release-89f59845afb19e89.yaml

```diff
@@ -0,0 +1,13 @@
+---
+prelude: >
+    The Qiskit Experiments 0.5 release added the visualization module for
+    drawing figures which replaces the previous plotting functionality. The
+    speed of randomized benchmarking experiments has been significantly
+    improved. The ``qubit`` and ``qubits`` input to experiments has been
+    regularized to ``physical_qubits``, and ``qiskit-ibmq-provider`` has been
+    deprecated in favor of ``qiskit-ibm-provider``. A new
+    :class:`.MultiStateDiscrimination` experiment has been added for classifying
+    IQ data. A test backend that uses Qiskit Dynamics for pulse-level
+    simulations has been added. The documentation has been significantly
+    refactored and introductory tutorials have been added. Various improvements
+    and bug fixes were made.
```

Comment:
I would move this last sentence to the first sentence like "The Qiskit Experiments 0.5 release brings various improvements and bug fixes. The highlights of this release include..." to hint to the reader that the rest of the paragraph is a collection of unrelated changes.